platform/osgi/platform-osgi-configadmin/src/main/java/org/codice/felix/cm/file/WrappedPersistenceManager.java

```diff
@@ -34,6 +34,13 @@
   }
 
   @Override
+  public void close() throws Exception {
+    if (persistenceManager instanceof WrappedPersistenceManager) {
+      ((WrappedPersistenceManager) persistenceManager).close();
```

Comment:
✏️ Alternatively, could check if `instanceof AutoCloseable`. On the off chance Karaf were to introduce a closeable persistence manager of its own in the future.

Reply:
❓ In that case, should I just do an `instanceof Closeable` in case they don't use the newer version?

Reply:
That's the way to do it. It will simply catch more (in the future) than simple testing if it is an instance of WrappedPersistenceManager

Reply:
Ah nevermind Closeable extends AutoCloseable not the other way around 👍